decidim-elections/app/models/decidim/elections/election.rb

```diff
@@ -23,6 +23,7 @@ class Election < ApplicationRecord
       has_many :questions, foreign_key: "decidim_elections_election_id", class_name: "Decidim::Elections::Question", inverse_of: :election, dependent: :destroy
       has_many :elections_trustees, foreign_key: "decidim_elections_election_id", dependent: :destroy
       has_many :trustees, through: :elections_trustees
+      has_many :votes, foreign_key: "decidim_elections_election_id", class_name: "Decidim::Elections::Vote", dependent: :destroy
```

Comment:
I think this will never happen anyway, but I'd say that is better to prevent the deletion if there is any vote related to it. So we can use `:restrict_with_exception` here, as explained [here](https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many-label-Options)